### ISSUE TITLE
Haskell.org homepage has poor accessibility

### ISSUE THREAD

**Issue by gwils** [haskell-infra/www.haskell.org#10]
I reckon haskell.org is not accessible. For example, if you load https://haskell.org on your desktop and keep pressing tab, focus becomes stuck in the "try ghci" window. You cannot proceed through the rest of the website by tabbing. This means screen reader users basically cannot use the website. I'm not sure what happens on mobile but I suspect it's similar.

I intend to learn more about accessibility and hopefully improve this. I know basically nothing about accessibility as is. I would welcome any volunteers who already know what they're doing.

**Comment by ketzacoatl** [comment]
Does anyone know what breaks or interferes with the cycling thru elements with tab? Is it that the "try ghci" window has javascript that consumes tab or other keypresses?
